mimir-build-image/build.sh

```diff
@@ -18,4 +18,7 @@ echo "weave:x:$uid:$gid::$SRC_PATH:/bin/sh" >>/etc/passwd
 echo "weave:*:::::::" >>/etc/shadow
```

Comment:
[unrelated] This made me realise we could change user name to "mimir" in a follow up PR :)